README.md

```diff
@@ -1,38 +1,50 @@
-jupyter-vim-binding
-========================
-Enable Vim key bindings in [jupyter](https://jupyter.org/) or [IPython notebook](http://ipython.org/notebook.html).
+#Jupyter VIM Binding
```

Comment:
I prefer to put an exact name of the repository (jupyter-vim-binding) as a title